### PR TITLE
Explore010

### PR DESCRIPTION
**Pull request by emilyashley**
closes 

- #542 
- #544 
- #493 
- #551 
- #545 
- #489 
- #543 
- #550 
- #488 
- #422

### CODE REVIEW

**Comment by emilyashley** [comment]
closes 

- #542 
- #544 
- #493 
- #551 
- #545 
- #489 
- #543 
- #550 
- #488 
- #422

File: mapstory/static/style/site/site.less [hunk 1]
```diff
@@ -1,6 +1,6 @@
 //fonts
-@import '../../fonts/lato_font.css';
 @import '../../site/font-awesome.css';
+@import 'https://fonts.googleapis.com/css?family=Lato:400,400i,700,900';
```
Comment (glynnis) [inline mapstory/static/style/site/site.less:3]: @cuttlefish Here's where Lato gets imported.

**Review by josellausas — DISMISSED** [review]
Fails to provision a fresh machine. 

**When:** 

`vagrant destroy && vagrant up --provision`

**I get this error:**

```
"stderr": "bower                               
ECONFLICT Unable to find suitable version for angular-aria", 

"stdout": "bower angular-animate#1.3.15       

not-cached https://github.com/angular/bower-angular-animate.git#1.3.15\nbower angular-animate#1.3.15          

resolve https://github.com/angular/bower-angular-animate.git#1.3.15\nbower angular-material#1.0.7
```

**Comment by emilyashley** [comment]
Here's my to-do of quirks to finish: 

- [x] check boxes don't sync with query events
- [ ] duplicate city names can appear in autocomplete results
- [ ] no visual feedback from fields that require a match to add a chip (storyteller & country & now, City)
- [x] not all usernames showing up in list of storytellers to search by on content page (try current user vs Capitalizedusername, etc)
- [x] first name autocomplete searches, last name autocomplete searches, but first last name autocomplete doesnt search
- [ ] pagination sometimes shows with no results switching from content to user explore


- [x] ensure grunt-concat makes it into deployment & static collection commands
- [x] check in about how to update owners api in our geonode

Update from @glynnis:
- [x] (works slow) Popular count (views) doesn't update on story content cards, but does on detail_page

- [ ]  VTC filter is angular, so it doesn't update already-rendered django counts of results
- [x] City field is case sensitive

**Comment by glynnis** [comment]
In reviewing the PR and setting up/adding users and data to my local environment, I came across some of the following issues:

* ~Profile pages aren't working. Whenever I create a new user and save changes to their profile (or click from Explore to go to a profile page), I see a blank page. To reproduce, get your local environment running and visit `http://192.168.56.151/storyteller/glynnis/`, and replace "glynnis" with the username of a user that exists on your local environment. Here's the error associated with it:~
~Error: [$injector:unpr] http://errors.angularjs.org/1.3.15/$injector/unpr?p0=DjangoConstantsProvider%20%3C-%20DjangoConstants%20%3C-%20Django
    at vendor-assets-min.js:17
    at vendor-assets-min.js:49
    at Object.d [as get] (vendor-assets-min.js:47)
    at vendor-assets-min.js:49
    at d (vendor-assets-min.js:47)
    at Object.e [as invoke] (vendor-assets-min.js:47)
    at Object.$get (vendor-assets-min.js:45)
    at Object.e [as invoke] (vendor-assets-min.js:47)
    at vendor-assets-min.js:49
    at d (vendor-assets-min.js:47)~
* ~Searching StoryTellers by City doesn't seem to do anything. For example, here I've entered "Washington" and pressed enter/pressed the search icon button, but no filter has been applied:~
[Outdated screenshot.](https://cloud.githubusercontent.com/assets/1529366/24514424/da7f80c0-1539-11e7-877a-d4576a2c3d63.png)~
* "Clear"/"Clear all" link not working on:
  * ~City field for StoryTellers~
  * ~Country field for StoryTellers~
  * ~Interest field for StoryTellers~
  * Content type checkboxes for Content (I think this is something you've already mentioned in your list above)
  * Category checkboxes for Content (I think this is something you've already mentioned in your list above)
  * ~StoryTeller for Content~
  * ~Tags for Content (it clears the popular tags that the user has selected, but not what's in the input box)~
* ~No autocomplete dropdowns are showing for me on the country or interest fields for StoryTellers, and nothing happens with them when I press enter or the search icon in those fields.~

**Comment by glynnis** [comment]
~Not sure if this is something you already captured in your list above, but I'm also not seeing chips display in the tags area of the Content tab when I select a popular tag. Here are some screenshots below, the first showing no popular tag selected, the next showing the tag selected and in the URL but not showing in the sidebar at all. The filtering and clearing in this case is working, it's just the display of the chip that seems to be broken.~

[Outdated screenshot.](https://cloud.githubusercontent.com/assets/1529366/24517777/258a4086-1545-11e7-9f08-93718a629528.png)
[Outdated screenshot.](https://cloud.githubusercontent.com/assets/1529366/24517776/25844604-1545-11e7-94a4-a91eb48a8fe7.png)

**Comment by glynnis** [comment]
~Not sure if this is related to this work or perhaps is a separate bug, but since it affects the "Popular" and "Newest" sorting dropdown on the Content page I'm reporting here (let me know if you'd prefer a separate bug report issue).~

~When I view another user's StoryLayer, I see the views increment on the StoryLayer page by 2 each time I refresh. However, when I go back to the Explore page and do a hard refresh, those views aren't reflected on the StoryLayer card (and thus don't apply and mean that the sorting dropdown does nothing).~

~[Outdated screenshot.](https://cloud.githubusercontent.com/assets/1529366/24518113/53ad3e2c-1546-11e7-8afd-fd00e85ffd33.png)
[Outdated screenshot.](https://cloud.githubusercontent.com/assets/1529366/24518114/53b25e02-1546-11e7-8991-6be7809b17d2.png)~

~I'll share a video of this on Slack so you can see better what I'm talking about.~

**UPDATE**: I'm seeing view counts on the cards now, there just seems to be a significant delay before new view counts are added. This also means that the dropdown for sorting on the content page is working now too :+1:

**Comment by glynnis** [comment]
I think this is something @emilyashley already looked into and it's difficult to achieve because of Django/Angular issues, but reporting here anyway in case there's a way to resolve: when you click "VTC Members", the "Showing 1-5 of 5 results" does not update the way it does for other results. "Showing" text should always reflect the numbers of what is displayed, not the original un-filtered query.

screen shot 2017-03-30 at 1 07 57 pm

**Comment by glynnis** [comment]
**UPDATE**: Emily and I were able to get on a call to troubleshoot my local environment (she'll follow up here with the magical incantations that were involved). I'm going through my prior comments now and adding a strike-through to any issues that have been resolved as a result of getting my local environment working.

**Comment by glynnis** [comment]
It looks like the city field is case sensitive, which I don't think it should be -- if as a user I enter "Boston" or "boston", I would expect to see the same results. See screenshots below for "Mainville" vs. "Mainville".
screen shot 2017-03-30 at 2 33 39 pm
screen shot 2017-03-30 at 2 33 47 pm

**Comment by emilyashley** [comment]
@Coop56 if I wanted to include `grunt concat` in the ansible or fab tasks how / where would I do that?

**Review by Coop56 — APPROVED** [review]
I've tested the code locally and everything appears to be working as intended.  More comments / documentation are needed but @emilyashley is working on those and we can pull them in a separate PR.  No reason to hold this one up.  Merging.

**Comment by emilyashley** [comment]
Thank you! :tophat:

**Pull request merged**